float32 wes
float32 wes_left
float32 turn
float32 angle1
float32 angle2
float32 angle3
int8    stoptrack
